src/main/java/school/hei/haapi/service/UserService.java

```diff
@@ -50,6 +51,12 @@ public void uploadUserProfilePicture(MultipartFile profilePictureAsMultipartFile
     fileService.uploadObjectToS3Bucket(bucketKey, savedProfilePicture);
   }
 
+  public User suspendStudentById(String suspendedStudentId) {
+    User suspendedStudent = findById(suspendedStudentId);
+    suspendedStudent.setStatus(SUSPENDED);
+    return userRepository.save(suspendedStudent);
```

Comment:
prefer single field update instead of updating all the fields

Reply:
i don't really understand can you explain more ? Like i must create a new method in repository who will update that field only or ?

Reply:
yes